JuiceMaker/JuiceMaker/Controller/FixStockViewController.swift

```diff
@@ -1,30 +1,119 @@
-//
-//  FixStockViewController.swift
-//  JuiceMaker
-//
-//  Created by 오승기 on 2021/06/10.
-//
 
 import UIKit
 
-class FixStockViewController: UIViewController {
+protocol FixStockViewControllerDelegate: class {
+    func fixStockViewControllerDidTapCloseButton(_ fixStockVC: FixStockViewController)
+}
 
-    @IBAction func closeButton(_ sender: UIBarButtonItem) {
-        dismiss(animated: true, completion: nil)
+class FixStockViewController: UIViewController {
+    
+    @IBOutlet weak var strawberryLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var kiwiLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    
+    @IBOutlet weak var strawberryStepper: UIStepper!
+    @IBOutlet weak var bananaStepper: UIStepper!
+    @IBOutlet weak var pineappleStepper: UIStepper!
+    @IBOutlet weak var kiwiStepper: UIStepper!
+    @IBOutlet weak var mangoStepper: UIStepper!
+    
+    var fruitStore: FruitStore?
+    weak var delegate: FixStockViewControllerDelegate?
+    
+    @IBAction func strawberryStepperValueChanged(_ sender: UIStepper) {
+        guard let fruit = fruitStore?.strawberry else {
+            return
+        }
+        tryStepper(fruit: fruit, sender: sender)
+        strawberryLabel.text = fruitStore?.strawberry.stock.description
+    }
+    @IBAction func bananaStepperValueChanged(_ sender: UIStepper) {
+        guard let fruit = fruitStore?.banana else {
+            return
+        }
+        tryStepper(fruit: fruit, sender: sender)
+        bananaLabel.text = fruitStore?.banana.stock.description
     }
-    @IBAction func starawberryStepper(_ sender: UIStepper) {
+    @IBAction func pineappleStepperValueChanged(_ sender: UIStepper) {
+        guard let fruit = fruitStore?.pineapple else {
+            return
+        }
+        tryStepper(fruit: fruit, sender: sender)
+        pineappleLabel.text = fruitStore?.pineapple.stock.description
     }
-    @IBAction func bananaStepper(_ sender: UIStepper) {
+    @IBAction func kiwiStepperValueChanged(_ sender: UIStepper) {
+        guard let fruit = fruitStore?.kiwi else {
```

Comment:
가드문 너무 잘써주셨어요!
다만 바인딩시에 변수명이 이왕이면 좀 더 명확하면 좋을 것 같아요 ㅎㅎ
```suggestion
        guard let kiwi = fruitStore?.kiwi else {
```